synapse/storage/databases/main/event_push_actions.py

```diff
@@ -458,57 +473,151 @@ def _get_thread(thread_id: str) -> NotifCounts:
         # updated `event_push_summary` synchronously when persisting a new read
         # receipt).
```

Comment:
We should update this comment to reflect that we now that we use threaded read receipts to determine whether an event is read

Reply:
Good call, I attempted this in e7b5421e47c174b139da2eef3e032d7a40f93cb2.